### ISSUE TITLE
data-processing enclave application crashes when decrypting text

### ISSUE THREAD

**Issue by HOGENTJasperDeSmet** [aws-samples/aws-nitro-enclaves-workshop#24]
Hi, First off all thank you for the excellent workshop. It really helped me understand the attestation process trough KMS

One remark
When trying to decrypt plaintext information through the enclave.
the enclave crashes with the following message.

`Traceback (most recent call last):
  File "/app/server.py", line 87, in <module>
    main()
  File "/app/server.py", line 73, in main
    plaintext = get_plaintext(credentials)
  File "/app/server.py", line 23, in get_plaintext
    creds = decrypt_cipher(access, secret, token, ciphertext, region)
  File "/app/server.py", line 51, in decrypt_cipher
    plaintext = json.loads(base64.b64decode(b64text))
  File "/usr/lib64/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.7/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 5 (char 4)
child exited with error`

As far as I know, this is because of line 52 in the server.py script. I don't really understand why the json.loads(plaintext) is necessary there so I just replaced it with plaintext.decode() and it works fine now.

**Comment by jdbean** [comment]
Hello! We are so glad to hear that you've found the workshop helpful and appreciate your feedback. Could you please confirm whether the Enclave application you are using is based on the most recent version of the sample application? There was a was a patch applied on July 26th 2021 which may be relevant to this issue.

**Comment by HOGENTJasperDeSmet** [comment]
I followed the instructions provided on [the workshop page](https://nitro-enclaves.workshop.aws/en/getting-started/prerequisites.html)
and executed the following command this morning
$ git clone --depth 1 https://github.com/aws-samples/aws-nitro-enclaves-workshop.git

Unfortunately, I have since deleted the demo instance so I cant double-check. 

Reading my original post and seeing that I did not quite explain how I got the error. I would like to add that I changed the values in the values.txt into words eg

"amazon
awsenclavedemo
demotest"

There is no problem with the original values.
